### ISSUE TITLE
bug in createCoordSeq

### ISSUE THREAD

**Issue by OmriRoames** [JuliaGeo/LibGEOS.jl#25]
`createCoordSeq` method doesn't work with am input vector

```julia
LibGEOS.createCoordSeq([0.0, 2.0])
```
results in

ERROR: UndefVarError: coord not defined
 in createCoordSeq(::Array{Float64,1}) at /home/omri/.julia/v0.5/LibGEOS/src/geos_functions.jl:161

**Comment by OmriRoames** [comment]
PR on its way

**Comment by OmriRoames** [comment]
Should we make it work with a vector of Int? i.e.
```julia 
LibGEOS.createCoordSeq([0, 2])
LibGEOS.createCoordSeq([[0, 2], [3,4]])
```
?

**Comment by yeesian** [comment]
I don't think we should promote the use of `createCoordSeq` (since they don't have many useful methods in LibGEOS, and are meant more for the construction of geometries), and should instead be promoting the use of the types in `src/geos_types.jl`

**Comment by yeesian** [comment]
closed by #26
